### PR TITLE
Update learning-tools.js to use correct crypto zombies url

### PR DESCRIPTION
**Pull request by rhigdon**
## Description

crypto zombies changed the url of their tutorial

### CODE REVIEW

**Comment by rhigdon** [comment]
could maybe get fancy with `useIntl` and properly direct to the proper launguage.  however, the languages are pretty well displayed on the landing page now

File: src/pages/developers/learning-tools.js [hunk 1]
```diff
@@ -119,7 +119,7 @@ const LearningToolsPage = ({ data }) => {
     {
       name: "CryptoZombies",
       description: "page-learning-tools-cryptozombies-description",
-      url: "https://cryptozombies.io/en/solidity",
+      url: "https://cryptozombies.io/",
```
Comment (rhigdon) [inline src/pages/developers/learning-tools.js:122]: Can also be `/en/course/`

**Comment by minimalsm** [comment]
> could maybe get fancy with `useIntl` and properly direct to the proper launguage. however, the languages are pretty well displayed on the landing page now

Nice idea! However, that breaks for languages that they do not support - (e.g. https://cryptozombies.io/sl/).

**Review by minimalsm — APPROVED** [review]
Thanks for the fix @rhigdon <3

Be sure to [join the discord](https://discord.gg/CetY6Y4) if you are interested in contributing further to the project or have any questions for the team. And we've just released our 2022 POAPs, so remember to [claim yours](https://ethereum.org/en/contributing/#poap) also 🥳!

**Comment by minimalsm** [comment]
@all-contributors please add @rhigdon for bugs, docs

**Pull request merged**